pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/troubleshoot/CommittingTransactionsCheckCommand.java

```diff
@@ -0,0 +1,314 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.tools.pravegacli.commands.troubleshoot;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import io.pravega.common.Exceptions;
+import io.pravega.controller.server.SegmentHelper;
+import io.pravega.controller.server.rpc.auth.GrpcAuthHelper;
+import io.pravega.controller.store.stream.StoreException;
+import io.pravega.controller.store.stream.StreamMetadataStore;
+import io.pravega.controller.store.stream.StreamStoreFactory;
+import io.pravega.controller.store.stream.VersionedMetadata;
+import io.pravega.controller.store.stream.records.*;
+import io.pravega.tools.pravegacli.commands.CommandArgs;
+import io.pravega.tools.pravegacli.commands.utils.CLIControllerConfig;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+
+import java.util.*;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static io.pravega.shared.NameUtils.computeSegmentId;
+import static io.pravega.shared.NameUtils.getEpoch;
+import static io.pravega.tools.pravegacli.commands.utils.CheckUtils.*;
+import static io.pravega.tools.pravegacli.commands.utils.OutputUtils.outputFaults;
+
+/**
+ * A helper class that checks the stream with respect to the committing_txn case.
+ */
+public class CommittingTransactionsCheckCommand extends TroubleshootCommandHelper implements Check{
+
+    /**
+     * Creates a new instance of the Command class.
+     *
+     * @param args The arguments for the command.
+     */
+    public CommittingTransactionsCheckCommand(CommandArgs args) { super(args); }
+
+    /**
+     * The method to execute the check method as part of the execution of the command.
+     */
+    @Override
+    public void execute() {
+        checkTroubleshootArgs();
+        try {
+            ScheduledExecutorService executor = getCommandArgs().getState().getExecutor();
+            store=createMetadataStore(executor);
```

Comment:
please apply checkstyle guidelines on the code.

Reply:
I have applied checkstyle guidelines